wix/Includes/OpenJDK.Variables.wxi

```diff
@@ -11,13 +11,13 @@
     <?define Arch="(x64)" ?>
   <?endif?>
 
-  <?define ProductVersion="$(var.ProductMajorVersion).$(var.ProductMinorVersion).$(var.ProductMaintenanceVersion).$(var.ProductPatchVersion)" ?>
+  <?define ProductVersion="$(var.MSIProductVersion)" ?>
```

Comment:
Sxa and I prefere to add BuildVersion here and not hide inside PatchVersion for better and for more detailed info
11.0.9.1+1
better than
11.0.9.101